portfolio/src/main/webapp/blog.html

```diff
@@ -21,22 +26,132 @@
         <a class="nav-link active " href="index.html">HOME</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link text-dark" href="about.html">ABOUT</a>
+        <a class="nav-link" href="about.html">ABOUT</a>
       </li>
       <li class="nav-item">
         <a class="nav-link" href="projects.html">PROJECT</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" href="blog.html">BLOG</a>
+        <a class="nav-link  text-dark" href="blog.html">BLOG</a>
```

Comment:
nit: remove extra space between "nav-link" and "text-dark"?

Reply:
Corrected!